server/http.go

```diff
@@ -119,7 +119,11 @@ func (p *Plugin) completeUserOAuthToZoom(w http.ResponseWriter, r *http.Request)
 	userID := stateComponents[1]
 	channelID := stateComponents[2]
 
-	p.API.KVDelete(state)
+	appErr = p.API.KVDelete(state)
+	if appErr != nil {
+		p.API.LogWarn("failed to delete state from db", "error", appErr.Error())
+		return
```

Comment:
Nit pick: I thinks it's fine to continue here

```suggestion
		p.API.LogWarn("failed to delete state from db", "error", appErr.Error())
```